python-sdk/langwatch/src/langwatch/utils/transformation.py

```
@@ -0,0 +1,218 @@
+        return super().default(o)
+
+
+class SerializableWithStringFallback(SerializableAndPydanticEncoder):
+    def default(self, o):
+        try:
+            if hasattr(o, "model_dump"):
+                return o.model_dump(exclude_unset=True)
+            if isinstance(o, set):
+                return list(o)
+            return super().default(o)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)